docs/server/cache.rst

```diff
@@ -26,19 +26,13 @@ Without a well functioning cache system, Pootle could be slow.
 
 Named Caches
 ------------
-The cache backends are at least configured with a 'default' cache.  If this is
-the only cache that exists then all caching is placed into this cache.  The
-Pootle cache can also be configured to take advantage of certain named caches.
+Pootle is configured with a these named caches:
 
-Current named caches:
-
-- ``'default'`` -- all non specified cache data and all cache data if only one
-  cache is defined.
+- ``'default'`` -- all non specified cache data and all cache data.
```

Comment:
The revision counter goes here too. This adds the risk of people clearing the default cache and also wiping out the revision counter — even if there's a way to restore it, we might want to prevent removal in the first place. I previously raised this concern in https://github.com/translate/pootle/issues/3500#issuecomment-73204160